feature-libs/product/configurators/rulebased/components/attribute/types/single-selection-bundle/configurator-attribute-single-selection-bundle.component.html

```diff
@@ -0,0 +1,25 @@
+<fieldset class="cx-configurator-attribute-single-selection-bundle">
+  <div class="cx-configurator-attribute-single-selection-bundle-quantity">
+    <div class="cx-single-selection-bundle-quantity-input">
+      <label>{{ 'configurator.attribute.quantity' | cxTranslate }}</label>
```

Comment:
from a11y perspective, label should be linked with an interactable element it belongs to. so try to figure out a way to do that somehow

Reply:
we use ItemCounterComponent, and this component dont have a 'id' attribute